blockdag/reachability.go

```diff
@@ -1013,19 +1020,16 @@ func (rt *reachabilityTree) expandIntervalInReindexRootChosenChild(reindexRoot *
 
```

Comment:
chosenReindexRootChild should be renamed to reindexRootChosenChild (like the call site)